src/fiatconnect/KycLanding.tsx

```diff
@@ -43,6 +45,16 @@ export interface Props {
 
 export default function KycLanding(props: StackScreenProps<StackParamList, Screens.KycLanding>) {
   const { quote, flow, step, personaKycStatus } = props.route.params
+  const personaInProgress = useSelector(personaInProgressSelector)
```

Comment:
If this state changes, I believe it cause the kycAgreement checkmark to be unset always. Should we pull the kycAgreement state to this component?

Reply:
Does it matter? Maybe ask Nitya to see if it's a consideration? Otherwise, I would vote to just keep it as it.

Reply:
chatted with Nitya, and updated to set the checkbox to true if step 1 is disabled